tests/infra/consortium.py

```diff
@@ -281,7 +281,7 @@ def _check_node_exists(self, remote_node, node_id, node_status=None):
         return True
 
     def wait_for_node_to_exist_in_store(
-        self, remote_node, node_id, node_status=None, timeout=3
+        self, remote_node, node_id, node_status=None, timeout=10
```

Comment:
Why the increased timeout? Is there a perf hit upgrading nlohmann?

Reply:
No, this is because I was trying to work out why #657 was happening, but I've found where that came from and I'll change that back.